benchmarks/ekobox/benchmark_evol_pdf.py

```diff
@@ -1,18 +1,17 @@
 # -*- coding: utf-8 -*-
 import pathlib
 
-import lhapdf
 import numpy as np
 import pytest
 
-import eko.output.legacy
 from eko import basis_rotation as br
 from ekobox import evol_pdf as ev_p
-from ekobox import gen_op as g_o
-from ekobox import gen_theory as g_t
+from ekobox import operators_card as oc
+from ekobox import theory_card as tc
 from ekobox.genpdf import load
 
 test_pdf = pathlib.Path(__file__).parent / "fakepdf"
+lhapdf = pytest.importorskip("lhapdf")
```

Comment:
I guess here you can even do
```suggestion
import lhapdf
```

It should be provided in the testing environment. However, even fine like this.

Reply:
let's keep like this